packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -607,8 +607,10 @@ class RawKeyboard {
       assert(mappedKeys != null,
         'Platform key support for ${Platform.operatingSystem} is '
         'producing unsupported modifier combinations.');
-      for (final PhysicalKeyboardKey physicalModifier in mappedKeys) {
-        modifierKeys[physicalModifier] = _allModifiers[physicalModifier];
+      if (mappedKeys != null) {
```

Comment:
There is an assert right above this line saying that mappedKeys should never be null. So, this `if` doesn't appear to be necessary?

/cc @gspencergoog

Reply:
If that assert is firing (or mappedKeys is `null` in release mode), then the `_modifierKeyMap` is probably missing a modifier key.

What modifier key is triggering the assert?